pkg/storage/s3/s3.go

```diff
@@ -75,3 +75,10 @@ func New(s3Conf *config.S3Config, timeout time.Duration, options ...func(*aws.Co
 		timeout:  timeout,
 	}, nil
 }
+
+func endpointFrom(credentialsEndpoint string, relativeURI string) string {
+	if relativeURI == "" {
+		return credentialsEndpoint
+	}
+	return credentialsEndpoint + relativeURI
```

Comment:
small nit, but always using `return credentialsEndpoint + relativeURI` should work in this case

Reply:
done! thanks!